emails/payin_bankwire_failed.spt

```diff
@@ -1,13 +1,13 @@
-{{ _("The bank wire to your Liberapay wallet has failed") }}
+{{ _("The bank wire to your Liberapay wallet failed") }}
 
 [---] text/html
 <p>{{ _(
-    "The attempt to transfer {0} from your bank account has failed with the following error message:",
+    "The attempt to transfer {0} from your bank account failed because:",
```

Comment:
Much shorter. Even "because" could be omitted here.

Reply:
No, you cannot use a colon just after `before`. See the punctuation rules on colons.

Reply:
Agreed, in between is in the middle of nowhere.